pulsar-broker/src/main/java/com/yahoo/pulsar/broker/PulsarService.java

```diff
@@ -95,6 +95,8 @@
     private LocalZooKeeperConnectionService localZooKeeperConnectionProvider;
     private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(20,
             new DefaultThreadFactory("pulsar"));
+    private final ScheduledExecutorService cacheExecutor = Executors.newScheduledThreadPool(10,
```

Comment:
just a thought - can we make the pools size configurable and put all constants in one place - for all pools cacheExecutor, scheduledExecutorScheduler and orderedExecutor